web_src/less/themes/theme-arc-green.less

```
@@ -1109,6 +1109,9 @@ input {
 .lines-code .hljs {
     background-color: #2a2e3a !important;
 }
+.blob-excerpt {
+    background-color: #252934 !important;
```

In combination with the other suggestion, `!important` is no longer required as this loads later and therefore takes precedence.
```suggestion
    background-color: #252934;
```
-----
Alternatively, up to other opinions as well but the color could mirror what the regular theme does by making the excerpt match the line number side-bar. See #10181 for a screenshot. Yours is more noticeable, though, so I have no strong opinion either way.
```suggestion
    background-color: #2e323e;
```

---

it was intended to make it noticable